Brightspace/insights-engagement-dashboard
# User filters - US121776

## Implementation Notes (optional)

Some interesting quirks of the structure is that the semester filter and the org filter selectors need to pass their event listeners up through the user drill page and then again to engagement. If anybody knows of a better way that would be 💯.

## Quality Notes

### Functional Testing (WIP)
 - Manual tests
   - When a semmester or orgunit is changed the records in the data object will change. This shows that the filters are being 
applied properly
 - Integration test
   - TODO check that something on the page or the data structures are changed as a result of a selection
### Performance - N/A

### Security - N/A

### Dev-Ops - N/A

### Cost - N/A

### UX
 - Requires demo to kevin

fyi @anhill-D2L @MykolaGalian @rohitvinnakota @johngwilkinson @bemailloux
